ansible/roles/hana_secondary_actions/tasks/main.yml

```diff
@@ -0,0 +1,183 @@
+---
+- name: Prepare secondary Action - Dump cluster status
+  command: crm status
+  become: true
+  register: crm_initial
+  changed_when: false
+
+- name: Prepare secondary Action - Assert this node is not master
+  assert:
+    that:
+      - crm_initial.stdout is not search('\* Masters:\s*\[\s*{{ inventory_hostname }}\s*\]')
+    fail_msg: "{{ inventory_hostname }} is MASTER – expected replica"
+    success_msg: "{{ inventory_hostname }} is in replica mode"
+
+- name: Prepare secondary Action - Wait cluster idle (pre-action)
+  command: cs_wait_for_idle --sleep 5
+  become: true
+  register: idle_pre
+  retries: "{{ cs_wait_timeout // 5 }}"
+  delay: 5
+  until: idle_pre.rc == 0
+
+# SBD-related stuff
+- name: Prepare secondary Action - Gather SBD-delay parameters
```

Comment:
is it not mostly the same code of https://github.com/SUSE/qe-sap-deployment/pull/367/files#diff-b434aa0a9d87f197afcd15b33e320b9c9f51001932a6cb13ad4a0f9c776bb506R32 ?

If it is so, suggestion in https://github.com/SUSE/qe-sap-deployment/pull/367/files#r2140352482 become even more relevant.